src/dmd/dsymbol.d

```diff
@@ -138,20 +138,20 @@ struct Prot
     Kind kind;
     Package pkg;
 
-    extern (D) this(Prot.Kind kind) pure nothrow @nogc @safe
+    extern (D):
+
+    this(Prot.Kind kind) pure nothrow @nogc @safe
     {
         this.kind = kind;
     }
 
-    extern (C++):
-
     /**
      * Checks if `this` is superset of `other` restrictions.
      * For example, "protected" is more restrictive than "public".
      */
-    bool isMoreRestrictiveThan(const Prot other) const
+    int opCmp(const Prot other) const pure nothrow @nogc @safe
```

Comment:
The comment above could better be explained with a ddoc-ed unittest.

Reply:
Good point, extended the documentation and added a documented unittest.